src/pages/index.js

```diff
@@ -3,13 +3,14 @@ import FormValidator from "../components/FormValidator";
 import Section from "../components/Section";
 import PopupWithForm from "../components/PopupWithForm";
 import PopupWithImage from "../components/PopupWithImage";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
 import Userinfo from "../components/Userinfo";
 import Card from "../components/Card";
 import Api from "../components/Api";
 import {
   initialCards,
   selectors,
-  config,
+  // config,
```

Comment:
Needs correction: please, make sure that the functionality for removing cards is working